### PR TITLE
docs: add FAQ section

### PR DESCRIPTION
**Pull request by portothree**
Hello, 

I was reading #162 and also found useful to have a FAQ section and have included the question mentioned in the issue as the first one. Hope this helps!

### CODE REVIEW

**Comment by rafaelss95** [comment]
Nice, @portothree... it will certainly help :)

File: README.md [hunk 1]
```diff
@@ -218,6 +219,11 @@ Contributions of any kind welcome!
 [**Read The Docs**](https://testing-library.com/angular) |
 [Edit the docs](https://github.com/testing-library/testing-library-docs)
 
+## FAQ
+
+##### I am using Reactive Forms and the `jest-dom` matcher `toHaveFormValues` always returns an empty object or there are missing fields. Why?
```
Comment (rafaelss95) [inline README.md:224]: How about mentioning the directive proposed by @timdeschryver?

Comment (timdeschryver) [reply README.md:224]: I'm going to merge this one already, because it helps other devs.
Feel free to update this and mention the directive @rafaelss95 .

**Review by timdeschryver — APPROVED** [review]

**Comment by timdeschryver** [comment]
Thanks @portothree !

@all-contributors please add @portothree  for docs

**Pull request merged**